src/namedtuples.jl

```diff
@@ -106,6 +106,14 @@ function rowtable(itr::T) where {T}
     return collect(namedtupleiterator(eltype(r), r))
 end
 
+function getrows(x::RowTable, inds; view::Union{Bool,Nothing} = nothing)
+    if view === true
+        return view(x, inds)
```

Comment:
```suggestion
        return Base.view(x, inds)
```
Because we have `view` as the keyword argument, we need to qualify our reference to the _function_. It'd be great to add some tests for this.